README.md

```diff
@@ -1,2 +1,12 @@
 # medicine_project
 Program to help you monitor your medication use
+
+## Setup
+
+1. Make sure you have Python set up, preferably in a [virtualenv](https://virtualenv.pypa.io/en/latest/user_guide.html)
```

Comment:
to jest wspaniała sprawa, z której warto korzystać przy każdym projekcie, bo wtedy nie musisz się przejmować tym czy zainstalowane paczki nadpisują czy psują inne paczki

Reply:
Jeszcze nie wiem co to paczki, ogólnie nie miej mi za złe jakbym odwlekał w czasie ogarnięcie czegoś z czym poleciłeś mi się zaznajomić. Sesja nadchodzi i w najbliższe tygodnie nie będę tu wiele robił.

Reply:
ło panie, ja tu niczego od ciebie nie wymagam :D Rób w swoim tempie, wszystko się z czasem objaśni.
Paczki to inaczej biblioteki, czyli to co se instalujesz za pomocą `pip install <coś>`. To twoje `icecream` jest paczką, jak też `pytest`

Reply:
Też ogólnie czytanie dokumentacji dla mnie jest dosyć ciężkie, tym bardziej po angielsku (czyli praktycznie każdej). Nie żebym nie chciał polecajek narzędzi, tylko nie zawsze będę czytał to od razu/całe 😅

Reply:
nie no, to jest całkowicie zrozumiałe. Dostajesz na raz milard rzeczy co masz znać - oczywiście że nie od razu będziesz wszystko wiedział, to jest kwestia praktyki. Ja ci będę mówił co i jak powinno być, ale to nie oznacza że musisz tak zrobić, a na pewno nie oznacza że masz to odrazu wiedzieć. Po to ci o tym tu piszę :D
Ja tu też daję różne odnośniki, zamiast sam tłumaczyć, bo nie ma sensu powtarzać to co już jest, ale też dlatego że to nie jest jakoś super istotne - to jest bardziej że warto to wiedzieć, ale są ważniejsze rzeczy

Reply:
ogólnie pisanie dokumentacji jest trudne. Przynjamniej jak ma być dobre. Większość dokumentacji jest do kitu, szczególnie jak to są małe projekciki. Także to całkiem normalne że ci się trudno czyta dokumentację. Osobiście często patrzę od razu do kodu, bo sam nie ogarniam o co autorowi chodziło